### ISSUE TITLE
[ui5-icon]: [The default mode "Image" of the icon produces a lot of a11y  Impact: serious errors]

### ISSUE THREAD

**Issue by veith** [SAP/ui5-webcomponents#10835]
### Bug Description

The new default for the attribute `mode` of the ui5-icon is the root cause for a lot of **a11y** errors in nearly every component in main and fiori which are using icons.  

Please consider to change this to `Decorative`. This should have no negative impact on existing code and represents the "reality" more precisely. 

In fiori and main are a lot of usages of the icon, some have the mode="Decoration" and a lot them have no mode set. There are only a few cases in the UI5 codebase where the `accessible-name` is set on the icons.

IMHO, there are only very few use cases where you want to use the icon as an image.

`<ui5-icon name="account" style="width:200px; height:200px" accessible-name="a person sitting in front of a pile of document"></ui5-icon>`

### Affected Component

progress indicator, tab container, calendar, shellbar,...

### Expected Behaviour

Using ui5 components should not trigger any a11y error.

### Log Output, Stack Trace or Screenshots

```bash
Error: Accessibility Violations
      ---
      Rule: svg-img-alt
      Impact: serious
      <svg> elements with an img role must have an alternative text (https://dequeuniversity.com/rules/axe/4.10/svg-img-alt?application=axeAPI)
      
      Issue target: ui5-progress-indicator,ui5-icon,svg
      Context: <svg class="ui5-icon-root" part="root" role="img" focusable="false" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg"><g role="presentation"></g></svg>
      Fix any of the following:
        Element has no child that is a title
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
      ---
```

### Priority

High

### UI5 Web Components Version

2.7.0

### Browser

Chrome

### Operating System

mac os

### Additional Context

Fixing every of the affected components could be an alternative, because as a user of the ui5 lib I have no possibility to do a workaround on the inner usages of the icon.

Running a tool like axe on every component in every theme could slightly improve the good a11y implementation of the ui5 components.  

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

It's true that the icon behaved as "decorative" by default in v1. We may change the default value to IconMode.Decorative 

Kind Regards,
Dobrin
